config/storage/backend/regex.json

```diff
@@ -50,14 +50,18 @@
       "@type": "DataAccessorBasedStore",
       "identifierStrategy": { "@id": "urn:solid-server:default:IdentifierStrategy" },
       "auxiliaryStrategy": { "@id": "urn:solid-server:default:AuxiliaryStrategy" },
-      "accessor": { "@id": "urn:solid-server:default:FileDataAccessor" }
+      "accessor": { "@id": "urn:solid-server:default:FileDataAccessor" },
+      "metadataStrategy":{ "@id": "urn:solid-server:default:MetadataStrategy" },
+      "converter": { "@id": "urn:solid-server:default:RepresentationConverter" }
     },
     {
       "@id": "urn:solid-server:default:MemoryResourceStore",
       "@type": "DataAccessorBasedStore",
       "identifierStrategy": { "@id": "urn:solid-server:default:IdentifierStrategy" },
       "auxiliaryStrategy": { "@id": "urn:solid-server:default:AuxiliaryStrategy" },
-      "accessor": { "@id": "urn:solid-server:default:MemoryDataAccessor" }
+      "accessor": { "@id": "urn:solid-server:default:MemoryDataAccessor" },
+      "metadataStrategy":{ "@id": "urn:solid-server:default:MetadataStrategy" },
+      "converter": { "@id": "urn:solid-server:default:RepresentationConverter" }
```

Comment:
I agree.

One important context element for this PR is that the plan is to open 2-3 issues after it to fix certain things that weren't done perfectly here because it would require a drastic change that would drastically increase the complexity of this PR. This is one of them.

On the other hand, perhaps I just needed a fresh look at it to come up with a clean solution for the problem, guess we'll see in the comments below. :smile: